modules/oadp-features-plugins-known-issues.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+// oadp-features-plugins-known-issues
+// * backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc
+
+:_content-type: CONCEPT
+[id="oadp-features-plugins-known-issues_{context}"]
+= OADP plugins known issues
+
+The following section describes known issues in {oadp-first} plugins:
+
+[id="velero-plugin-panic_{context}"]
+== Velero plugin panics on imagestream backup due to a missing secret
```

Comment:
The words _panics on_ sounds odd. I might be right for all I know. Would _panics during_ be correct? I think we need an article, too. 

```suggestion
== Velero plugin panics during an imagestream backup due to a missing secret
```

```suggestion
== Velero plugin panics during imagestream backups due to a missing secret
```